README.md

```diff
@@ -39,351 +39,358 @@ Works with any PostgreSQL database, including Timescale Cloud, Amazon RDS, Supab
 pip install pgai
 ```              
 
+
 # Quick Start
 
-This section will walk you through the steps to get started with pgai and Ollama using docker and show you the major features of pgai. 
+This quickstart demonstrates how pgai Vectorizer enables semantic search and RAG over PostgreSQL data by automatically creating and synchronizing embeddings as data changes.
+
+The key "secret sauce" of pgai Vectorizer is its declarative approach to
+embedding generation. Simply define your pipeline and let Vectorizer handle the
+operational complexity of keeping embeddings in sync, even when embedding
+endpoints are unreliable. You can define a simple version of the pipeline as
+follows:
+
+```sql
+SELECT ai.create_vectorizer(
+     'wiki'::regclass,
+     loading => ai.loading_column(column_name=>'text'),
+     destination => ai.destination_table(target_table=>'wiki_embedding_storage'),
+     embedding => ai.embedding_openai(model=>'text-embedding-ada-002', dimensions=>'1536')
+    )
+```
 
-Please note that using Ollama requires a large (>4GB) download of the docker image and model. If you don't want to download so much data, you may want to use the [OpenAI quick start](/docs/vectorizer/quick-start-openai.md) or [VoyageAI quick start](/docs/vectorizer/quick-start-voyage.md) instead.
+The vectorizer will automatically create embeddings for all the rows in the
+`wiki` table, and, more importantly, will keep the embeddings synced with the
+underlying data as it changes.  **Think of it almost like declaring an index** on
+the `wiki` table, but instead of the database managing the index datastructure
+for you, the vectorizer is managing the embeddings. 
 
-### Setup
+## Running the quick start
 
-Install PostgreSQL and Ollama if you don't already have them. 
+**Prerequisites:**
+- A PostgreSQL database (click here for docker instructions).
+- An OpenAI API key (we use openai for embedding in the quick start, but you can use [multiple providers](#supported-embedding-models)).
 
-1. **Download the [docker compose file](/examples/docker_compose_pgai_ollama/docker-compose.yml) file.**
+Create a `.env` file with the following:
 
-    ```
-    curl -O https://raw.githubusercontent.com/timescale/pgai/main/examples/docker_compose_pgai_ollama/docker-compose.yml
-    ```
+```
+OPENAI_API_KEY=<your-openai-api-key>
+DB_URL=<your-database-url>
+```
 
-1. **Start the docker compose file.**
-    ```
-    docker compose up -d
-    ```
+run the following code in the same directory as the `.env` file:
+
+<details>
+<summary>Click to expand code</summary>
+
+```python
```

Comment:
Wouldn't make sense to move all this python code into an actual .py file so people can download it (curl) ?